recipes/wg-blimp/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
   run:
     - click
     - h5py
+    - mamba
```

Comment:
Why does this need mamba as a run-time requirement?

Reply:
We are using Snakemake environments for each part of our pipeline. On our HPC systems we receive the error:

```
Error: mamba package manager is not available. The mamba package manager (https://github.com/mamba-org/mamba) is an extremely fast and robust conda replacement. It is the recommended way of using Snakemake's conda integration. It can be installed with `conda install -n base -c conda-forge mamba.If you still prefer to use conda, you can enforce that by setting `--conda-frontend conda`.
```
It seems that even setting the conda frontend to 'conda' does not fix this error (through Snakemake python API). I'm still trying to figure out why exactly that happens, but installing mamba resolved the issue, so I though the easiest way to fix the error would be to simply include it here. It does feel a little bit hacky though.

Reply:
@johanneskoester Do we need to add mamba as a run-time dependency to snakemake?

@MarWoes Normally you would install all of your dependencies as part of the recipe. Do you have a bunch of different environment for different steps?

Reply:
Possibly related: https://github.com/snakemake/snakemake/issues/939

Reply:
@dpryan79 We used to have all dependencies referenced in the recipe, but since there were so many the solving took forever and was at last also pulling in ancient versions of multiple packages (causing openssl errors for samtools for example). So we now use Snakemake environments for each step which drastically increased speed, stability and allows to use recent versions for all dependencies.

Reply:
`conda` is an optional dependency for Snakemake. Given that it's common for Conda packages to depend on optional dependencies, one would expect `snakemake` to depend on `conda`. However, IIRC, this is intentionally not done so that the user do not get a duplicate Conda installation when they `conda create --name=snakemake snakemake` a new environment (since the `envs/snakemake/bin/conda` doesn't necessarily "know" its own environment is not the actual `base` environment and will happily create its own package cache etc.).
As for `mamba`: `conda` is a dependency of `mamba` so the same thing as above applies.

Reply:
Thanks for your comments! Does this imply `mamba` should never be depended on when using `snakemake`? Or could I use this for the `wg-blimp` package if convenience might outweigh duplicate installations?

Reply:
If you're going to add it, make sure that users are aware that they can accidentally create sub-environments.